### PR TITLE
Skip empty lines instead of erroring

### PR DESCRIPTION
**Pull request by ThrawnCA**
Configure Tabulator to ignore empty rows, #206

### CODE REVIEW

File: ckanext/xloader/tests/samples/sample_with_empty_lines.csv [hunk 1]
```diff
@@ -0,0 +1,8 @@
+date,temperature,place
+2011-01-01,1,Galway
+2011-01-02,-1,Galway
+2011-01-03,0,Galway
+2011-01-01,6,Berkeley
+,,Berkeley
+2011-01-03,5,
+
```
Comment (duttonw) [inline ckanext/xloader/tests/samples/sample_with_empty_lines.csv:8]: please test with 2 new lines as single new line is standard.

File: ckanext/xloader/tests/test_loader.py [hunk 2]
```diff
@@ -632,6 +632,18 @@ def test_with_blanks(self, Session):
...
+        csv_filepath = get_sample_filepath("sample_with_empty_lines.csv")
+        resource = factories.Resource()
+        resource_id = resource['id']
+        loader.load_csv(
+            csv_filepath,
+            resource_id=resource_id,
+            mimetype="text/csv",
+            logger=logger,
+        )
+        assert len(self._get_records(Session, resource_id)) == 6
```
Comment (duttonw) [inline ckanext/xloader/tests/test_loader.py:645]: can we assert if direct load ran vs backfill tabulator.

**Review by duttonw — CHANGES_REQUESTED** [review]
please add one more test to verify behaviour of a blank line of 'commas' i.e
"a","b","c"
"a",,
,,,
"b","d","d"
,,,
,,,

File: ckanext/xloader/tests/samples/sample_with_empty_lines.csv (hunk 1 above)
Comment (ThrawnCA) [reply ckanext/xloader/tests/samples/sample_with_empty_lines.csv:9]: Added.

File: ckanext/xloader/tests/test_loader.py (hunk 2 above)
Comment (ThrawnCA) [reply ckanext/xloader/tests/test_loader.py:645]: Not relevant to unit tests. The fallback logic is in jobs.py, unit tests bypass that to directly call load_csv or load_table.

**Comment by ThrawnCA** [comment]
> please add one more test to verify behaviour of a blank line of 'commas' i.e "a","b","c" "a",, ,,, "b","d","d" ,,, ,,,

That's outside the scope of this PR; it's not really fixable as part of a direct COPY.

**Review by duttonw — APPROVED** [review]

**Pull request merged**